### PR TITLE
JUP-49 improve club contacts

### PR DESCRIPTION
**Pull request by nl32**
this pr adds:
- valid email support.
- url specific validation for all options.

### CODE REVIEW

**Review by RubenOlano — APPROVED** [review]
Looks good 👍🏽

File: src/utils/contact.ts
Comment (RubenOlano) [inline src/utils/contact.ts:1]: Recheck some of the regex for this. Our instagram link is considered invalid, I think you might be missing the www part

**Review by RubenOlano — CHANGES_REQUESTED** [review]
Discord and email validation seems to work fine

**Review by danielsuhcompsci — APPROVED** [review]
I think everything checks out 👌

File: src/utils/contact.ts
Comment (nl32) [reply src/utils/contact.ts:1]: just decided to remove regex validation, also going to be a nightmare.

**Pull request merged**